src/main/java/me/totalfreedom/totalfreedommod/command/Command_wipecoreprotectdata.java

```diff
@@ -17,11 +17,11 @@ public boolean run(final CommandSender sender, Player playerSender, Command cmd,
     {
         if (!plugin.cpb.isEnabled())
```

Comment:
This entire command probably needs deleting... Really not happy with the idea of admins being able to delete coreprotect data.

Reply:
done